# A simple cyclic test: go from the current position to the target pose, then go back and repeat.

cyclic_test: 

  urdf_path: $(find awesome_leg_pholus)/description/urdf/awesome_leg.urdf

  approaching_traj: # configurations for the node used to move the leg to its initial position

        # xbot2 control modes codes (bitmask, p-v-e-k-d->[1,2,4,8,16], where p is the LSB); to combine control mode, simply sum the single control codes
        # impedance (pos+kp+kd) + torque f.f. code: -> 29
        # simple impedance (pos+kp+kd) code: -> 25

        n_intervals: 200 # number of intervals of the generated forward trajectory
        T_execution: 3.0 # desired execution time (forward trajectory)
        standby_time: 0.0 # time delay between the finished approach maneouver and the start of the real trajectory

        q_p_target: [-1.57, 1.57] # target joint pose 
        
        ctrl_mode: [25, 25]
        stiffness: [500, 50]
        damping:  [5, 5]